cmd/fyne/internal/commands/get.go

```diff
@@ -10,6 +10,9 @@ import (
 
 	"github.com/urfave/cli/v2"
 	"golang.org/x/sys/execabs"
+
+	//lint:ignore SA1019 The recommended replacement does not solve the use-case
+	"golang.org/x/tools/go/vcs"
```

Comment:
I understand what the ignore comment is saying but I'm a bit worried about starting to import a module that never will get updated any more. Is there not a better package we can use? Or some other way to do it?

Reply:
The only other way to do it would be to copy a load of code out of the go/cmd/go internal module resolver code.
See the ongoing feature request at https://github.com/golang/go/issues/18387

Reply:
Alright. That makes a lot of sense. I was just wondering if there was some other third-party package that did the same thing. If that's not the case then I think your approach makes complete sense.

Reply:
Alright. I see that now. Will test the code later today and approve if everything looks good :)